packages/react/src/components/listBox/ListBox.tsx

```diff
@@ -151,12 +151,12 @@ export class ListBox extends Component<IListBoxProps> {
     render() {
         const items = this.props.isLoading ? this.getLoadingItems() : this.getItems();
         return (
-            <>
+            <div className={classNames({'list-box-with-footer': this.props.footer})}>
```

Comment:
Do you really need this div? You already have the with-footer class on the ul when you have a footer

Reply:
It's the way I found to change the bottoms corners from curved to flat to it's straight lines between the list and the footer 😅

Reply:
In that case it would be the other way around, you should keep the div but remove the `with-footer` class on the ul since you target the element by using: `.list-box-with-footer .list-box` and style it accordingly

Reply:
Done! Thanks for the tip! 😄